distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -1110,7 +1110,7 @@ Vector.Vector.write_bytes : (File|Text) -> Existing_File_Behavior -> Nothing ! I
 Vector.Vector.write_bytes self path on_existing_file=Existing_File_Behavior.Backup =
     Panic.catch Unsupported_Argument_Types_Data handler=(Error.throw (Illegal_Argument_Error_Data "Only Vectors consisting of bytes (integers in the range from -128 to 127) are supported by the `write_bytes` method.")) <|
         ## Convert to a byte array before writing - and fail early if there is any problem.
-        byte_array = Array_Utils.ensureByteArray self.to_array
+        byte_array = Array_Builder.ensureByteArray self.to_array
```

Comment:
Okay in the changeset I did not notice that there's also this function. Indeed that one would fit more in a `Array_Utils` module IMO. But `Array_Builder` looks fine too I guess.

Reply:
Thank you. I tried to convert the method into an _instance method_:
```
byte[] toByteArray() { ... }
```
but then I'd have to handle the conversion manually. I decided to leave that for later.

Reply:
ok